deploy/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -438,6 +440,15 @@ spec:
           - get
           - list
           - watch
+        - apiGroups:
+          - ocs.openshift.io
+          resources:
+          - storagerequests
+          verbs:
+          - delete
+          - get
+          - list
+          - watch
```

Comment:
Shouldn't we be removing any RBAC related to storage request instead of adding?

Reply:
Earlier it was clubbed with other resources here which had different set of permissions https://github.com/red-hat-storage/ocs-operator/pull/3070/files#diff-d277c2c7cd2cad5b3a3f064a324edc09356ebbc04577d53afc8b37df6b3f08b0L389. Now it does not need all thoose persmissions hence controller-gen has separtly added the entry with only required verbs.